### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by tantommy**
## Description

- adds e2e tests for playwright + browserstack local integration
- adds circleCI config.yml
- adds husky to run e2e tests locally on playwright emulators pre-push

## Related Issue

Fixes https://github.com/liftedinit/albert/issues/69
Fixes https://github.com/liftedinit/albert/issues/68

### CODE REVIEW

**Comment by tantommy** [comment]
merge https://github.com/liftedinit/albert/pull/67 first

**Comment by hansl** [comment]
Added @fmorency for reviewing the `config.yml`. I'm going to finish reviewing the tests themselves soon.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          command: npm run test
+      - run:
+          name: Build app
+          command: npm run build
+
+  create:
+    executor: node/default
+    steps:
+      - checkout
+      - run: install-packages
```
Comment (fmorency) [inline .circleci/config.yml:30]: Remove `run: `

**Review by fmorency — COMMENTED** [review]
Thanks, Tommy!

**Review by fmorency — COMMENTED** [review]
I would also pin the node version(s) you're testing on.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+commands:
+  install-packages:
+    description: install packages
+    steps:
+      - node/install-packages:
+          pkg-manager: npm
+      - run:
+          name: Update many-js
+          command: npm update many-js
```
[the remark this answers is not in the document]
Comment (tantommy) [reply .circleci/config.yml:128]: this will be removed once many-js ci process is in place bc then we'll have a consistent publish to npm registry whereas right now the source is from github.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+          command: |
+            node --version
+            npm --version
+      - install-packages
+      - run:
+          name: Run tests
+          command: npm run test
+      - run:
+          name: Build app
+          command: npm run build
```
[the remark this answers is not in the document]
Comment (tantommy) [reply .circleci/config.yml:24]: looked at `run_tests_with_npm` but not really necessary as the config is essentially the same, if not more.
looked at `node_npm_run` and this one we could probably change to once many-js CI is merged and we re-install many-js to a proper version in alberto.  that could be another pr.

File: .circleci/config.yml (hunk 1 above)
Comment (tantommy) [reply .circleci/config.yml:30]: done

**Comment by tantommy** [comment]
> I would also pin the node version(s) you're testing on.

done.  it should take the node version from `.nvmrc` which now has a specific node version

**Comment by hansl** [comment]
@tantommy can we test against multiple versions in a matrix? I want LTS and latest at least.

**Comment by tantommy** [comment]
> @tantommy can we test against multiple versions in a matrix? I want LTS and latest at least.

matrix of `lts`, `current`, and `16.16.0` has been added.

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+          name: Run tests
+          command: npm run test
+      - run:
+          # node 17+ workaround https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported
+          name: set openssl-legacy-provider in NODE_OPTIONS
+          command: |
+              if [ "<< parameters.node_version >>" == "current" ]
+              then
+                echo 'export NODE_OPTIONS=--openssl-legacy-provider' >> $BASH_ENV
+              fi
```
Comment (fmorency) [inline .circleci/config.yml:34]: Please use a `when` directive instead of doing the check here.

File: README.md [hunk 5]
```diff
@@ -13,5 +13,24 @@ npm install
...
+  - `MAIN_WALLET_PEM` should be the PEM file of the address with initial tokens
+    found in `ledger_state.json5` in many-framework.
+  ```
+  initial: {
+      "maffbahksdwaqeenayy2gxke32hgb7aq4ao4wt745lsfs6wijp": {
+          "MFX": 1000000000
+      }
+  },
+  ```
+  - Add the three seed phrses
```
Comment (fmorency) [inline README.md:31]: `phrases`.

We already have some testing accounts in the genesis file, can we use those?

Same with our dev `id1.pem` file, could we provide it to users/dev?

I think users and devs should be able to build/run our tests without intervention.

This could be another PR, however.

WDYT?

**Review by fmorency — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 4 above)
Comment (tantommy) [reply .circleci/config.yml]: when directive is now used.

File: README.md (hunk 5 above)
Comment (tantommy) [reply README.md]: default values for pem and seed phrases have been added.

Comment (fmorency) [reply README.md]: @hansl Pinging you here following our 1:1.

**Review by fmorency — APPROVED** [review]
